db/seeds.rb

```diff
@@ -1,7 +1,13 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+50.times do
+  adj = Faker::Hacker.adjective
+  noun = Faker::Hacker.noun
+  color = Faker::Commerce.color
+  title = "#{adj} #{noun} #{color}"
+  paragraph_count = rand(3) + 1
+  desc = Faker::Lorem.paragraph paragraph_count
+  image = Faker::Avatar.image
+
+  Book.create title: title, description: desc, cover_image_url: image
+end
```

Comment:
Can you elaborate a little? I'm not sure I follow you: build an array of all the 50 elements and then create them all at once?